tycho/routes/event.py

```diff
@@ -137,6 +140,7 @@
     :return: the stored event
     """
     await request.app["db"].event.save(event)
+    LOG.info(event.to_primitive())
```

Comment:
could this be a configuration parameter? just thinking people may not want to log every event normally.

Then again the volume is pretty low.